OrbitCore/FunctionFramepointerValidator.cpp

```diff
@@ -47,6 +55,10 @@ bool FunctionFramepointerValidator::ValidatePrologue() {
   // check the first instruction: it must be "push ebp"
   if (cs_regs_access(handle_, &instructions_[0], regs_read, &read_count,
                      regs_write, &write_count) == 0) {
+    if (instructions_[0].id == X86_INS_ENTER) {
+      return true;
+    }
```

Comment:
ok, this probably introduces another state... the question here is, is it valid to use ENTER without LEAVE and LEAVE without ENTER.

Reply:
As discussed offline, this would be strange hand-crafted assembly, which would silently accept here.